projecttacoma/fqm-testify
# Upload page

# Summary
Moved the measureUpload and datePicker components to a separate page

## New Behavior
datePicker and Measure Bundle upload are now on a separate page
A measure must be uploaded (and dates picked) before navigating to test resource creation
The measure Bundle upload page can be returned to by clicking the edit button on the test resource creation page

## Code Changes
Created new `generateTestCases` page
Broke resource creation out of index and added it to generateTestCases Page
Added measure Bundle & measurement period display to generateTestCases page in line with Mockups

# Testing Guidance
- `npm run check`
- `npm run dev`
- Note the `next` button is disabled until a measure is uploaded
- Upload a measure and click next
- Ensure the bundle and period display is accurate and in line with mockup
- Add test resources
- Head back to measure upload by clicking the edit icon
- Head back to test resource creation and ensure the data persists
